data/ecosystem/vendors.yaml

```diff
@@ -41,6 +41,13 @@
   contact: ''
   oss: false
   commercial: true
+- name: Cribl
+  distribution: false
+  nativeOTLP: true
+  url: 'https://docs.cribl.io/stream/sources-otel'
+  contact: ''
```

Comment:
please add a point of contact for us (ideally a github handle, can be an email address)

Reply:
Updated with contact information - hello@cribl.io